components/director/cmd/main.go

```diff
@@ -53,6 +56,8 @@ type config struct {
 	JWKSSyncPeriod      time.Duration `envconfig:"default=5m"`
 	AllowJWTSigningNone bool          `envconfig:"default=true"`
 
+	StaticUsersSrc string `envconfig:"default=/data/static-users.yaml"`
```

Comment:
Please modify `run.sh` script and provide a valid path to static-users.yaml for local development.